src/main/scala/no/ndla/frontpageapi/service/ConverterService.scala

```diff
@@ -127,11 +139,45 @@ object ConverterService {
     }
   }
 
+  def toDomainSubjectPage(toMergeInto: domain.SubjectFrontPageData,
+                          subject: api.UpdatedSubjectFrontPageData): Try[domain.SubjectFrontPageData] = {
+
+    val domainLayout = subject.layout
+      .map(toDomainLayout)
+
+    val domainBannerImage = subject.banner
+      .map(toDomainBannerImage)
+
+    val domainMetaDescription = subject.metaDescription.fold(Seq[MetaDescription]())(toDomainMetaDescription)
+
+    val domainAboutSubject =
+      subject.about.fold(Seq[AboutSubject]())(about => toDomainAboutSubject(about).getOrElse(Seq()))
+
+    Try(
```

Comment:
Syns det er slightly tydeligere å bruke `Success` dersom denne copy statementen ikke kan kaste en exception :)
```suggestion
    Success(
```